### PR TITLE
Update rs.m3u

### PR DESCRIPTION
**Pull request by Carlinhos027**

### CODE REVIEW

File: streams/rs.m3u [hunk 1]
```diff
@@ -1,54 +1,56 @@
...
-#EXTINF:-1 tvg-id="",Muzzik Worldwide Serbia (720p) [Geo-blocked]
+#EXTINF:-1 tvg-id="WORLDWIDEMUZZIK.rs",Worldwide Muzzik (720p) 
 https://muzzik-live.morescreens.com/mts-5/playlist.m3u8
-#EXTINF:-1 tvg-id="NisvilleTV.rs",Nisville TV RS (720p)
-http://92.60.237.32:1935/live/nisville/playlist.m3u8
-#EXTINF:-1 tvg-id="NTV.rs",NTV (576p)
-https://vod1.laki.eu/ntv/video.m3u8
+#EXTINF:-1 tvg-id="NisvilleTV.rs",Nisville TV (1080p)
+https://tv.nisville.com/live/nisville/playlist.m3u8
+#EXTINF:-1 tvg-id="Pink.rs",Pink (720p)
```
Comment (BellezaEmporium) [inline streams/rs.m3u:52]: + https://tv.liveott.online/PINK_M.m3u8

File: streams/rs.m3u [hunk 2]
```diff
@@ -75,27 +77,17 @@ http://109.105.201.198/PINKWORLD/playlist.m3u8
...
-#EXTINF:-1 tvg-id="",Prva Srpska TV (576p)
-http://14.202.204.202:8000/play/a00i/index.m3u8
+#EXTINF:-1 tvg-id="Prva.rs",Prva Srpska TV (720p)
+https://tv.liveott.online/PRVA.m3u8
 #EXTINF:-1 tvg-id="RedTV.rs",Red TV (720p)
 https://live.rednet.rs/providus/redtv_multi.m3u8
 #EXTINF:-1 tvg-id="RedTV.rs",Red TV (720p)
 https://live.rednet.rs/providus/redtv_multi_hq/index.m3u8
 #EXTINF:-1 tvg-id="RTS1.rs",RTS 1 (720p) [Not 24/7]
 https://webtvstream.bhtelecom.ba/rts1.m3u8
```
Comment (BellezaEmporium) [inline streams/rs.m3u:87]: + https://tv.liveott.online/RTS_1.m3u8
+ https://tv.liveott.online/RTS_2.m3u8

**Comment by BellezaEmporium** [comment]
Uses an apparent sub from https://mojasupernova.rs / https://supernovaott.rs

**Comment by Carlinhos027** [comment]
> Uses an apparent sub from https://mojasupernova.mts.rs / https://supernovaott.rs

Can them be added then?

**Comment by BellezaEmporium** [comment]
> > Uses an apparent sub from https://mojasupernova.mts.rs / https://supernovaott.rs
> 
> Can them be added then?

I do not see an apparent issue, but i do not know if it'll stay long or not.

**Review by BellezaEmporium — DISMISSED** [review]

**Review by Alstruit — DISMISSED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**